src/sections/greenNav.html

```diff
@@ -0,0 +1,37 @@
+<div
+  class="bg-primaryColor fixed top-0 z-10 flex w-full justify-between p-7 text-primaryGreen"
+>
+  <div class="text-smallerSize flex flex-col md:text-2xl">
+    <h1>DE</h1>
+    <h1>EN</h1>
+  </div>
+  <div class="relative text-3xl font-light lg:text-5xl">
+    <img
+      src="/src/sections/img/Vector-5.webp"
+      alt="menu-image"
+      class="h-14 w-48 md:hidden"
+    />
+    <p
+      class="text-smallSize absolute left-4 top-2 font-semibold md:relative md:text-5xl lg:text-5xl"
+    >
+      Baumeister
+    </p>
+  </div>
+  <div class="hidden md:block">
+    <div class="flex flex-col text-2xl">
+      <a href="#about">About</a>
```

Comment:
You can leave it by now because you will need to navigate to the corresponding sections later